test/integration/taproot.spec.ts

```diff
@@ -528,6 +528,70 @@ describe('bitcoinjs-lib (transaction with taproot)', () => {
     });
   });
 
+  it('can create (and broadcast via 3PBP) a taproot script-path spend Transaction - OP_CHECKSIGADD (2-of-3) and verify unspendable internalKey', async () => {
+    const leafKeys = [];
+    const leafPubkeys: Buffer[] = [];
+    for (let i = 0; i < 3; i++) {
+      const leafKey = bip32.fromSeed(rng(64), regtest);
+      leafKeys.push(leafKey);
+      leafPubkeys.push(toXOnly(leafKey.publicKey));
+    }
+
+    // The only thing that differs between the wallets is the private key.
+    // So we will use the first wallet for all the Psbt stuff.
+    const [wallet, wallet2, wallet3] = leafKeys.map(key =>
+      new TaprootMultisigWallet(
+        leafPubkeys,
+        2, // Number of required signatures
+        key.privateKey!,
+        LEAF_VERSION_TAPSCRIPT,
+      ).setNetwork(regtest),
+    );
+
+    // amount from faucet
+    const amount = 42e4;
+    // amount to send
+    const sendAmount = amount - 1e4;
+    // get faucet
+    const unspent = await regtestUtils.faucetComplex(wallet.output, amount);
+
+    const psbt = new bitcoin.Psbt({ network: regtest });
+
+    // Adding an input is a bit special in this case,
+    // So we contain it in the wallet class
+    // Any wallet can do this, wallet2 or wallet3 could be used.
+    wallet.addInput(psbt, unspent.txId, unspent.vout, unspent.value);
+
+    psbt.addOutput({ value: sendAmount, address: wallet.address });
+
+    // Sign with at least 2 of the 3 wallets.
+    // Verify that there is a matching leaf script
+    // (which includes the unspendable internalPubkey,
+    // so we verify that no one can key-spend it)
+    wallet3.verifyInputScript(psbt, 0);
+    wallet2.verifyInputScript(psbt, 0);
+    psbt.signInput(0, wallet3);
+    psbt.signInput(0, wallet2);
+
+    // Before finalizing, we need to add dummy signatures for all that did not sign.
+    // Any wallet can do this, wallet2 or wallet3 could be used.
+    wallet.addDummySigs(psbt);
```

Comment:
Taproot's OP_CHECKSIGADD method of multisig requires that all keys that do not sign must have an empty signature (`[] === OP_FALSE === 0x00`).

That's why this needs to be done.

Reply:
Sorry, i think only signTaprootInputWithDummy is needed

Reply:
I don't think this is necessary. However, I think I can make it clearer in this example how to deal with the problem.

Reply:
OK, let`s add example first

Reply:
Creating a dummy signer doesn't work because updateInput checks to see if the signature length is 64 or 65.

We need the dummy signatures to be length 0.

So just using signInput with a Signer that returns an empty Buffer for signing will not work.

That said, this is an extremely niche use case, and adding a proper method to handle this edge case is not necessary.